webpack/dlls/README.md

```diff
@@ -0,0 +1,31 @@
+# DLLs
+
+Webpack "DLLs" can help reduce the build times a bit which is rather useful
+during development.
+
+You may opt-in to use the Vendor DLL by doing the following:
+
+- set `WEBPACK_DLLS=1` in your shell before running webpack
+- compile the DLL using `npm run build-dlls`
```

Comment:
This is a little ambiguous, am I supposed to run `WEBPACK_DLLS=1 npm run dev` ? Is it only for development? Should there be specific npm tasks for running webpack with this env set?